src/main/java/com/mathworks/ci/MatlabBuildStepExecution.java

```diff
@@ -122,6 +121,11 @@ private void createMatlabScriptByName(FilePath uniqueTmpFolderPath, String uniqu
         if (!tasks.trim().isEmpty()) {
             cmd += " " + tasks;
         }
+
+        if (!buildOptions.trim().isEmpty()) {
+            cmd += " " + buildOptions;
```

Comment:
It would be good to check when build options are passed without - dash prefixed does it show the appropriate Error message. Do we have any Integ test for this ? @Vahila

Reply:
That's a good case to verify. The integ tests for build Options are not added yet.